docs/integration/alerts-spikesh-integration.md

```diff
@@ -0,0 +1,50 @@
+title: Spike.sh Alerts Integration
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with Spike.sh. Use them together to get the information about each notification quickly and on any device.
+
+## In Spike.sh
+
+**1.** Create an account at [Spike.sh](https://spike.sh) if you don't have one already.
+
+To be able to receive incoming notifications we need to add and configure a new integration. To do that:
```

Comment:
To be able to receive incoming notifications we need to add and configure a new integration. To do that:
==> 
To receive incoming notifications from Sematext add and configure a new integration in Spike.sh:

Reply:
Adjusted.